clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -201,7 +230,11 @@ func (s *schedulerLatencyMeasurement) setQuantileFromHistogram(metric *measureme
 		if err != nil {
 			return err
 		}
-		metric.SetQuantile(quantile, time.Duration(int64(histQuantile*float64(time.Second))))
+		// NaN is returned only when there are less than two buckets.
+		// In which case all quantiles are NaN and all latency metrics are untouched.
+		if !math.IsNaN(histQuantile) {
```

Comment:
so if a metric is extremely stable and all data points fall in one bucket, we will see nothing in the dashboard?

Reply:
If a metric is given only with a single bucket, then sadly yes. Though, as long as the only bucket with samples is not the first bucket , there always going to be at least two buckets.

Reply:
In case someone decides to omit buckets with no samples, the histogram code can be extended for this special case. As long as the upper bound is not +Inf.

Reply:
I don't understand why ```hist.Quantile``` doesn't handle this case already, but I guess this is not something we can fix in this PR because this is vendored code.